Sources/MySQL/Database.swift

```diff
@@ -90,24 +131,59 @@ public final class Database {
             May be empty if the call does not produce data.
     */
     @discardableResult
-    public func execute(_ query: String, _ values: [NodeRepresentable] = [], _ on: Connection? = nil) throws -> [[String: Node]] {
-        // If not connection is supplied, make a new one.
-        // This makes thread-safety default.
-        let connection: Connection
-        if let c = on {
-            connection = c
-        } else {
-            connection = try makeConnection()
+    public func execute(_ query: String, _ values: [NodeRepresentable] = []) throws -> [[String: Node]] {
```

Comment:
Do we need this function if the above can run w/ default args? Not sure the difference why we need a unique signature is clear offhand.